### PR TITLE
feat: support swagger configuration

### PR DESCRIPTION
**Pull request by easonlin404**
now URL variable can be set.

address https://github.com/swaggo/swag/issues/194

### CODE REVIEW

**Comment by easonlin404** [comment]
please DONOT merge, I will add testings.

**Comment by pei0804** [comment]
Destructive change, is it okay?
https://travis-ci.org/swaggo/swag/jobs/510253209

**Comment by pei0804** [comment]
@easonlin404

File: swagger.go [hunk 1]
```diff
@@ -11,21 +11,28 @@ import (
...
 
+// Config stores ginSwagger configuration variables.
+type Config struct {
+	//The url pointing to API definition (normally swagger.json or swagger.yaml).
+	URL string
+}
+
 // WrapHandler wraps `http.Handler` into `gin.HandlerFunc`.
-func WrapHandler(h *webdav.Handler) gin.HandlerFunc {
+func WrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
```
Comment (pei0804) [inline swagger.go]: It seems safer to prepare another function.

Comment (pei0804) [reply swagger.go]: I think feel good is revert and create another function.

Comment (easonlin404) [reply swagger.go]: I am syncing  up with swag code. sorry about bother.

Comment (pei0804) [reply swagger.go]: I see.
I was concerned about was the impact on users already using `WrapHander` .
So I think create another interface. eg: config setable and default config (`WrapHander`) .

Comment (easonlin404) [reply swagger.go]: I agree. could you help add this feat?

Comment (easonlin404) [reply swagger.go]: @pei0804

Comment (pei0804) [reply swagger.go]: OK I do it.
Before that, I would like to revert this PR once to minimize the impact.

Comment (easonlin404) [reply swagger.go]: ok

**Pull request merged**